packages/kbn-text-based-editor/src/text_based_languages_editor.test.tsx

```diff
@@ -163,60 +154,6 @@ describe('TextBasedLanguagesEditor', () => {
     ).toBe(0);
   });
 
-  it('should  render the errors badge for the inline mode by default if errors are provided', async () => {
```

Comment:
ℹ️ The deleted tests are related to the compact mode which is entirely removed in this redesign